### PR TITLE
Feature 99

### PR DESCRIPTION
**Pull request by arpitNEC**
please review and merge.

### CODE REVIEW

File: Core/AtContextServer/src/main/java/eu/neclab/ngsildbroker/atcontextserver/config/AtContextServerResourceConfigurer.java [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
 import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
 import eu.neclab.ngsildbroker.commons.securityConfig.ResourceConfigDetails;
+import eu.neclab.ngsildbroker.commons.swaggerConfig.SwaggerConfigDetails;
```
Comment (amitrnec) [inline Core/AtContextServer/src/main/java/eu/neclab/ngsildbroker/atcontextserver/config/AtContextServerResourceConfigurer.java:12]: Please remove. It is unused package

File: Core/EntityManager/src/main/java/eu/neclab/ngsildbroker/entityhandler/config/EntityManagerResourceConfigurer.java [hunk 2]
```diff
@@ -3,13 +3,15 @@
...
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
 import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
 
 import eu.neclab.ngsildbroker.commons.securityConfig.ResourceConfigDetails;
+import eu.neclab.ngsildbroker.commons.swaggerConfig.SwaggerConfigDetails;
```
Comment (amitrnec) [inline Core/EntityManager/src/main/java/eu/neclab/ngsildbroker/entityhandler/config/EntityManagerResourceConfigurer.java:14]: Please remove. It is unused package

File: Core/QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/config/QueryManagerResourceConfigurer.java [hunk 3]
```diff
@@ -2,12 +2,14 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
 import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
 import eu.neclab.ngsildbroker.commons.securityConfig.ResourceConfigDetails;
+import eu.neclab.ngsildbroker.commons.swaggerConfig.SwaggerConfigDetails;
```
Comment (amitrnec) [inline Core/QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/config/QueryManagerResourceConfigurer.java:12]: Please remove. It is unused package

File: Core/SubscriptionManager/src/main/java/eu/neclab/ngsildbroker/subscriptionmanager/config/SubscriptionManagerResourceConfigurer.java [hunk 4]
```diff
@@ -2,12 +2,14 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
 import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
 import eu.neclab.ngsildbroker.commons.securityConfig.ResourceConfigDetails;
+import eu.neclab.ngsildbroker.commons.swaggerConfig.SwaggerConfigDetails;
```
Comment (amitrnec) [inline Core/SubscriptionManager/src/main/java/eu/neclab/ngsildbroker/subscriptionmanager/config/SubscriptionManagerResourceConfigurer.java:12]: Please remove. It is unused package

File: History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/config/HistoryManagerResourceConfigurer.java [hunk 5]
```diff
@@ -2,12 +2,14 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
 import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
 import eu.neclab.ngsildbroker.commons.securityConfig.ResourceConfigDetails;
+import eu.neclab.ngsildbroker.commons.swaggerConfig.SwaggerConfigDetails;
```
Comment (amitrnec) [inline History/HistoryManager/src/main/java/eu/neclab/ngsildbroker/historymanager/config/HistoryManagerResourceConfigurer.java:12]: Please remove. It is unused package

File: Registry/RegistryManager/src/main/java/eu/neclab/ngsildbroker/registryhandler/config/RegistryManagerResourceConfigurer.java [hunk 6]
```diff
@@ -2,12 +2,14 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
 import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
 import eu.neclab.ngsildbroker.commons.securityConfig.ResourceConfigDetails;
+import eu.neclab.ngsildbroker.commons.swaggerConfig.SwaggerConfigDetails;
```
Comment (amitrnec) [inline Registry/RegistryManager/src/main/java/eu/neclab/ngsildbroker/registryhandler/config/RegistryManagerResourceConfigurer.java:12]: Please remove. It is unused package

**Review by ScorpioBroker — APPROVED** [review]
LGTM

**Pull request merged**